SPECS/python-distlib/python-distlib.signatures.json

```diff
@@ -0,0 +1,5 @@
+{
+    "Signatures": {
+     "distlib-0.3.6.tar.gz": "14bad2d9b04d3a36127ac97f30b12a19268f211063d8f8ee4f47108896e11b46"

```

Comment:
Not 100% sure, but I thought our blob tarballs were supposeds to use the package name (in this case, python-distlib)

Reply:
The most general guideline is "make sure the name is clear enough to understand, which package it goes to, and that there's little to no risk of collisions". In this case, "distlib" may be somewhat risky, if there's a non-Python version of this package, so I'd also suggest a re-name.